rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/google/GCEBakeHandlerSpec.groovy

```diff
@@ -159,6 +160,9 @@ class GCEBakeHandlerSpec extends Specification {
                                         base_os: "ubuntu",
                                         cloud_provider_type: BakeRequest.CloudProviderType.gce)
       def targetImageName = "kato-x8664-timestamp-ubuntu"
+      def osPackages = bakeRequest.package_name.split(" ").collect { packageName ->
```

Comment:
This pattern appears enough that it might make sense to extract it into a utility function for use in all the unit tests.